Documentation/cli-installation-scenarios.md

```diff
@@ -242,8 +262,16 @@ OSX/Linux:
 
 #### Windows obtain one-liner example 
 
+CMD:
+
+```
+@powershell -NoProfile -ExecutionPolicy unrestricted -Command "&{iwr https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scripts/obtain/dotnet-install.ps1 | iex}"
+```
+
+PowerShell:
+
 ```
-@powershell -NoProfile -ExecutionPolicy unrestricted -Command "&{iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scripts/obtain/dotnet-install.ps1'))}"
+iwr https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scripts/obtain/dotnet-install.ps1 | iex
```

Comment:
BTW, running dotnet-install.ps1 is throwing error at the moment:

```powershell
The variable '$Stream' cannot be retrieved because it has not been set.
At line:1 char:1
```